docs/userguide/configuration/execution.rst

```diff
@@ -1,235 +0,0 @@
-.. _label-execution:
-
-Execution
-=========
-
-Contemporary computing environments may include a wide range of computational platforms or **execution providers**, from laptops and PCs to various clusters, supercomputers, and cloud computing platforms. Different execution providers may require or allow for the use of different **execution models**, such as threads (for efficient parallel execution on a multicore processor), processes, and pilot jobs for running many small tasks on a large parallel system. 
-
-Parsl is designed to abstract these low-level details so that an identical Parsl program can run unchanged on different platforms or across multiple platforms. 
-To this end, Parsl uses a configuration file to specify which execution provider(s) and execution model(s) to use.
-Parsl provides a high level abstraction, called a *block*, for providing a uniform description of a compute resource irrespective of the specific execution provider.
```

Comment:
This section appears to be removed; I think there are a couple of gems in here worth saving.  For example, onboarding new users involves a fair amount of general playing-field education, and I think this serves some of that purpose.